Libraries/Image/Image.ios.js

```diff
@@ -62,7 +62,7 @@ var Image = React.createClass({
     /**
      * `uri` is a string representing the resource identifier for the image, which
      * could be an http address, a local file path, or the name of a static image
-     * resource (which should be wrapped in the `required('image!name')` function).
+     * resource (which should be wrapped in the `require('image!name')` function).
```

Comment:
@amasad/@frantic: when are we going to update documentation and examples for the new relative path hotness?  Would be good to do it earlier rather than later to reduce the amount of legacy people build up in their apps.